install_requirements.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
```

Comment:
I mentioned it above. It does not seem worth adding that install script when all is needed is to execute 3 lines to clone and install dependency.